src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,16 @@
+import { useLocation } from "react-router-dom";
+
+function ErrorPage({ data }) {
+
+    const location = useLocation()
+
+    return (
+        <section id="error-notification">
+            <h1>Oops...</h1>
+            <h2>We can't seem to find what you're looking for.</h2>
+            <p>Error message: {data ? data.message : location.state.message}</p>
+        </section>
+    )
+}
+
```

Comment:
Great use of a reusable `ErrorPage` component that takes in an argument to provide the error message 👍